src/Lhm/Lhm.php

```diff
@@ -152,12 +154,12 @@ public static function cleanup($run = false, array $options = [])
                 $adapter->query("DROP TRIGGER IF EXISTS {$trigger}");
             }
 
+            $adapter->query('SET foreign_key_checks=0');
```

Comment:
The actual LHM code is: https://github.com/soundcloud/lhm/blob/8d6e8fdfa3b2ca0be1d719d8720759eaec193d65/lib/lhm.rb#L59